### PR TITLE
feat(TDI-44106): add possibility to select fields to be returned.

### PR DESCRIPTION
**Pull request by dmytrogrygorenko**
**Please check if the PR fulfills these requirements**

- [X] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [X] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-44106
Added possibility to select fields to be returned by Input components, to avoid MongoDB error 'The reply message length 4297571 is less than the maximum message length 4194304'
Depends on https://jira.talendforge.org/browse/TDI-45590

**What is the new behavior?**

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.components.bigdata/components/tCosmosDBInput/tCosmosDBInput_messages.properties [hunk 1]
```diff
@@ -55,6 +55,9 @@ QUERY_SPECS.NAME = Query
 QUERY_TYPE.NAME = Query Type
 QUERY_TYPE.ITEM.AGGREGATION_QUERY = Aggregation Pipeline Query
 QUERY_TYPE.ITEM.FIND_QUERY = Find Query
+SPECIFY_FIELDS.NAME=Specify fields to return
+FIELDS_TO_RETURN.NAME=Return only this fields:
+FIELDS_TO_RETURN.ITEM.FIELD_NAME=Field Name
```
Comment (mbasiuk-talend) [inline main/plugins/org.talend.designer.components.bigdata/components/tCosmosDBInput/tCosmosDBInput_messages.properties:60]: Lower case the second word.

**Review by mbasiuk-talend — APPROVED** [review]
Except for the formatting, looks good to me.

**Review by mbasiuk-talend — APPROVED** [review]
LGTM. The only thing to check the correct wording for new properties.

File: main/plugins/org.talend.designer.components.bigdata/components/tCosmosDBInput/tCosmosDBInput_messages.properties (hunk 1 above)
Comment (dmytrogrygorenko) [reply main/plugins/org.talend.designer.components.bigdata/components/tCosmosDBInput/tCosmosDBInput_messages.properties]: Done

**Review by mbasiuk-talend — DISMISSED** [review]

**Review by mbasiuk-talend — APPROVED** [review]

**Review by chmyga — APPROVED** [review]
LGTM

**Pull request merged**